tests/helpers/assertions.py

```diff
@@ -53,3 +53,33 @@ def has_log_message(output, level="info", message=""):
 
 def udp_port_open_locally(port):
     return os.system("cat /proc/net/udp | grep %s" % (hex(port)[2:].upper(),)) == 0
+
+
+# check the fake backend if any metric in `metrics` exist
+def any_metric_found(backend, metrics):
```

Comment:
`backend` -> `fake_services` to match the rest of the assertion functions.

Reply:
Also, this whole function can just be `return any([has_datapoint_with_metric_name(backend, m) for m in metrics])`